### PR TITLE
Updating with latest from Definitely Typed and snippets repo

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5a82697](https://github.com/OfficeDev/office-js-docs-reference/pull/443/commits/5a82697748117893399111eeb48e353b3a907858)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfillloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfillloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfont.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfont.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfontloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfontloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.comment.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.comment.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.commentcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.commentcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.commentreply.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.commentreply.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customfunctioneventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.customfunctioneventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.datavalidation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.datavalidation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.eventtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.eventtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplycollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplycollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplydata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplydata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplyloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplyloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplyupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplyupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangecollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangecollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangeloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangeloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicercollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicercollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tablecollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tablecollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tabledata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tabledata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tableloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tableloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tablescopedcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tablescopedcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tableupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tableupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotfield.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotfield.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotfieldcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotfieldcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivothierarchy.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivothierarchy.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotitemcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotitemcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.rangeareas.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.rangeareas.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.rowhiddenchangetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.rowhiddenchangetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.settablecellproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.settablecellproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.slicer.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.slicer.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.sliceritem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.sliceritem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.tablechangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.tablechangedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetchangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.worksheetchangedeventargs.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201907032126135680-443%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1dtmFmzrNX1oassriL3tLBBJfhPXXP0NBGM7g2V3b%2Bk%3D&st=2019-07-03T21%3A27%3A49Z&se=2019-08-03T21%3A32%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: generate-docs/json/snippets.yaml [hunk 1]
```diff
@@ -15447,6 +15447,225 @@ Excel.Binding.onDataChanged:
...
+                    color: true
+                },
+                font: {
+                    color: true
+                }
+            },
+            style: true
+        });
+
+        // Sync to get the data from the workbook
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/json/snippets.yaml]:
```suggestion
        // Sync to get the data from the workbook.
```

File: generate-docs/json/snippets.yaml [hunk 2]
```diff
@@ -17215,14 +17434,111 @@ Excel.Range.style:
...
+    });
 Excel.RangeAreas.format:
   - |-
     await Excel.run(async (context) => {
 
         const sheet = context.workbook.worksheets.getActiveWorksheet();
         const usedRange = sheet.getUsedRange();
 
-        // Find the ranges with formulas.
+        // Find the ranges with formulas
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/json/snippets.yaml]:
```suggestion
        // Find the ranges with formulas.
```

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. I'm fixing those suggestions in the upstream snippets: https://github.com/OfficeDev/office-js-snippets/pull/324

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9a89bb7](https://github.com/OfficeDev/office-js-docs-reference/pull/443/commits/9a89bb7920cbff7b678a640a596a27a6338497a2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfill.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfill.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfillloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfillloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfont.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfont.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesfontloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesfontloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.cellpropertiesloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.cellpropertiesloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.comment.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.comment.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.commentcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.commentcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.commentreply.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.commentreply.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customfunctioneventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.customfunctioneventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.datavalidation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.datavalidation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.eventtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.eventtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplycollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplycollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplydata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplydata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplyloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplyloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentreplyupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentreplyupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.commentupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.commentupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangecollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangecollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.rangeloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.rangeloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicercollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicercollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemdata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemdata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.sliceritemloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.sliceritemloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.slicerupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.slicerupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tablecollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tablecollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tabledata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tabledata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tableloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tableloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tablescopedcollectionloadoptions.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tablescopedcollectionloadoptions.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.interfaces.tableupdatedata.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.interfaces.tableupdatedata.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotfield.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotfield.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotfieldcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotfieldcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivothierarchy.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivothierarchy.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.pivotitemcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.pivotitemcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.rowhiddenchangetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.rowhiddenchangetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.settablecellproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.settablecellproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.slicer.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.slicer.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.sliceritem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.sliceritem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.tablechangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.tablechangedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetchangedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.worksheetchangedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DT/docs/docs-ref-autogen/excel/excel.worksheetcollection.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201907032201255834-443%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SaPCRQ5N%2B1oNxSE26pJYE4%2FbU8oJkppfX87kXgq34BQ%3D&st=2019-07-03T22%3A03%3A13Z&se=2019-08-03T22%3A08%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**